gap-analysis/adlm-gap.html

```diff
@@ -149,6 +139,20 @@ <h3>Prioritization</h3>
 
 <p>A cell can be scored as OK if the feature in question is specified in an appropriate specification, and is supported by user agents. A specification that is in CR or later and has two implementations in 'major' browsers will count. This means that the feature may not be supported in all browsers yet. (At some point in the future we may try to distinguish, visually, whether support is available in a specification but still pending in major browsers or applications.) </p>
 </section>
+
+
+
+
+
+
+
+<section id="related">
+<h3>Related documents</h3>
+
+<p>A summary of this report and others can be found as part of the <a href="https://www.w3.org/International/typography/gap-analysis/language-matrix.html" style="font-size: 140%;">Language Matrix</a>.</p>
```

Comment:
I feel that the size of this is too different from the words next to it. For me, 135% is enough.